_posts/2020-04-05-error-handling.md

```diff
@@ -165,14 +165,14 @@ So where does that stand?
 * Configuration is very much about user-provided callbacks. So we are in the
   open-errors area.
 * A lot of these errors are going to be shown to the end user, so they have to
-  be nice and *meaningfull*. That means having enough of context for the user to
+  be nice and *meaningful*. That means having enough of context for the user to
   figure out what went wrong.
 
 At that time, the [failure](https://crates.io/crates/failure) appeared and it
 was the perfect tool for the job, because:
 
 * It has its `failure-derive` sub-crate (enabled as a feature). This cuts down
-  on the boilerplate of leaf errors. Just throw in few derive and annotation
+  on the boilerplate of leaf errors. Just throw in a few derive and annotation
```

Comment:
Note that `few` and `a few` are different things. In particular, `few` denotes smaller quantity than `a few`. Why do you think the latter is more appropriate?

Reply:
You have a good point, allow me to revert this change